### PR TITLE
feat: add support for Furiosa RNGD accelerator

### PR DESCRIPTION
**Pull request by rapsealk**
Follow-up of #1702.

This PR makes Furiosa RNGD accelerator to work with Backend.AI WebUI.

Ref: https://furiosa.ai/rngd

### CODE REVIEW

**Comment by yomybaby** [comment]
* **#2776** <a href="https://app.graphite.dev/github/pr/lablup/backend.ai-webui/2776?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @rapsealk and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: src/components/backend-ai-agent-summary-list.ts [hunk 1]
```diff
@@ -825,6 +843,30 @@ export default class BackendAIAgentSummaryList extends BackendAIPage {
...
             : html``}
+          ${rowData.item.rngd_slots
+            ? html`
+                <div
+                  class="layout horizontal center-justified flex progress-bar-section"
+                >
+                  <div class="layout horizontal start resource-indicator">
+                    <img
+                      class="indicator-icon fg green"
+                      src="/resources/icons/tpu.svg"
```
Comment (agatha197) [inline src/components/backend-ai-agent-summary-list.ts]:
```suggestion
                      src="/resources/icons/furiosa.svg"
```
how about using furiosa logo rather then tpu?

**Review by agatha197 — CHANGES_REQUESTED** [review]
I updated i18n and added a minor suggestion. Please check it. Thank you :)

File: src/components/backend-ai-agent-summary-list.ts (hunk 1 above)
Comment (rapsealk) [reply src/components/backend-ai-agent-summary-list.ts]: Thank you! I missed it.

**Review by agatha197 — APPROVED** [review]
LGTM

**Review by yomybaby — APPROVED** [review]
LGTM

**Pull request merged**